cli/src/commands/working-groups/verifyValidator.ts

```diff
@@ -0,0 +1,49 @@
+import WorkingGroupsCommandBase from '../../base/WorkingGroupsCommandBase'
+import { WorkingGroups } from '../../Types'
+import { VerifyValidator, RemarkMetadataAction } from '@joystream/metadata-protobuf'
+import { metadataToString } from '../../helpers/serialization'
+
+export default class VerifyValidatorAccountCommand extends WorkingGroupsCommandBase {
+  static description = 'Membership lead/worker verifies validator membership profile'
+  static args = [{
+    name: "memberId",
+    required: true,
+    description: 'Membership ID of the validator to verify',
+  },
+  {
+
+    name: "isVerified",
+    required: true,
+    description: 'Verification state of the validator',
+  }
+  ]
```

Comment:
Please use flags `instead` of `args`. Using flags you have to explicitly write each flag name before providing the value, so less chance of errors, as compared to `args` which are just order dependant positional arguments